### ISSUE TITLE
Unknown flag - storage class

### ISSUE THREAD

**Issue by eddiewebb** [CircleCI-Public/aws-eks-orb#10]
https://circleci.com/gh/CircleCI-Public/circleci-demo-aws-eks/44

```

Error: unknown flag: --storage-class
Usage: eksctl create cluster [flags]


```

**Comment by eddiewebb** [comment]
I think it must be CLI rev, removed from a dev version of orb and it gets past that.

**Comment by lokst** [comment]
Thanks @eddiewebb , it seems that the flag was removed as part of https://github.com/weaveworks/eksctl/pull/1029 because it was used for 1.10 k8s clusters and according to https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html:`Amazon EKS will deprecate Kubernetes version 1.10 on July 22, 2019. On this day, you will no longer be able to create new 1.10 clusters, and all Amazon EKS clusters running Kubernetes version 1.10 will be updated to the latest available platform version of Kubernetes version 1.11. `

**Comment by lokst** [comment]
Fixed in 0.2.0 of the orb, thanks @eddiewebb !
